# SSL_DetectionFrame.msg

Header header


uint32             frame_number 
double             t_capture     
double             t_sent        
uint32             camera_id     
SSL_DetectionBall  balls         
SSL_DetectionRobot robots_yellow 
SSL_DetectionRobot robots_blue   
